app/lib/helper.js

```diff
@@ -0,0 +1,27 @@
+const mkdirp = require('mkdirp');
+
+function createDirectory(path) {
+  mkdirp(path);
+}
```

Comment:
You can simply do:

```js
const createDirectory = require('mkdirp')
```

but I suggest using the name `mkdirp` because it more accurately signals what it does, i.e. `mkdir -p`, which creates a directory and all its parent directories (if they don't already exist).

Reply:
`mkdirp` sounds awesome to me ^^